### PR TITLE
Adding basic formatting for output types.

### PR DESCRIPTION
**Pull request by swernli**
@jstarks @jterry75 
Added a Docker.Format.ps1xml that includes some starting formatting for Containers, Images, and Networks when output to the cmdline.
This addresses https://github.com/Microsoft/Docker-PowerShell/issues/34

### CODE REVIEW

File: src/Docker.PowerShell/Cmdlets/GetContainerNetwork.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 namespace Docker.PowerShell.Cmdlets
 {
     [Cmdlet(VerbsCommon.Get, "ContainerNetwork")]
-    [OutputType(typeof(NetworkResponse))]
+    [OutputType(typeof(NetworkListResponse))]
```
Comment (jterry75) [inline src/Docker.PowerShell/Cmdlets/GetContainerNetwork.cs]: You need both right? The same cmdlet returns both depending on list or inspect?

File: src/Docker.PowerShell/Docker.Format.ps1xml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  <View>
+    <Name>Container</Name>
+    <ViewSelectedBy>
+      <TypeName>Docker.DotNet.Models.ContainerListResponse</TypeName>
+    </ViewSelectedBy>
+    <TableControl>
+      <TableHeaders>
+        <TableColumnHeader>
+          <Label>ID</Label>
+          <Width>20</Width>
```
Comment (jterry75) [inline src/Docker.PowerShell/Docker.Format.ps1xml]: How much of the ID does docker CLI show in chars?

File: src/Docker.PowerShell/Cmdlets/GetContainerNetwork.cs (hunk 1 above)
Comment (jstarks) [inline src/Docker.PowerShell/Cmdlets/GetContainerNetwork.cs]: It sounds like we should split the cmdlet, then.

File: src/Docker.PowerShell/Docker.Format.ps1xml (hunk 2 above)
Comment (jstarks) [inline src/Docker.PowerShell/Docker.Format.ps1xml]: 12

File: src/Docker.PowerShell/Docker.Format.ps1xml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+              <PropertyName>ID</PropertyName>
+            </TableColumnItem>
+            <TableColumnItem>
+              <PropertyName>Image</PropertyName>
+            </TableColumnItem>
+            <TableColumnItem>
+              <PropertyName>Command</PropertyName>
+            </TableColumnItem>
+            <TableColumnItem>
+              <ScriptBlock>[DateTime]::Now - [TimeSpan]$_.Created</ScriptBlock>
```
Comment (jstarks) [inline src/Docker.PowerShell/Docker.Format.ps1xml]: Does this actually work? When I tried a private earlier, this resulted in incorrect times.

File: src/Docker.PowerShell/Cmdlets/GetContainerNetwork.cs (hunk 1 above)
Comment (swernli) [inline src/Docker.PowerShell/Cmdlets/GetContainerNetwork.cs]: I'll make that split in this change.

File: src/Docker.PowerShell/Docker.Format.ps1xml (hunk 3 above)
Comment (swernli) [inline src/Docker.PowerShell/Docker.Format.ps1xml]: Turns out there is a missing special handler in the generated Docker.DotNet code.  I'll update this to just be a propertyname in anticipation of the update to Docker.DotNet.

**Comment by swernli** [comment]
@jstarks Ok, made the updates based on feedback.  Try it out...

**Comment by jstarks** [comment]
Get-ContainerNetworkDetail is missing from the psd1...

**Comment by jstarks** [comment]
LGTM

**Pull request merged**